values.yaml

```diff
@@ -79,7 +84,11 @@ auth:
     enabled: false
     host: ldap://<hostname>:<port>
     searchBase: CN=Users,DC=example,DC=com
-    searchFilter: CN=john
+    admin: cn=admin,dc=example,dc=com
+    searchFilter: (objectClass=*)
+    UserIdentityAttribute: cn
```

Comment:
Should this variable be lower camel case like all the other ones?

Reply:
I confirm @AyadiAmen SSL, UserIdentity, AuthStrategy, IdentityStrategy, AuthExpiration should be lower case.